<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Головне вікно</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Працівники</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>541</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="HireButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Найняти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MoveButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Перевести</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FireButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Звільнити</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистити пошук</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>541</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_findname"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_findsurname"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_findposition"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_find">
         <property name="text">
          <string>Пошук</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Відсутність</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>541</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addAbsence">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Відмітити</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_report_Button">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Звіт</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Департамент</string>
     </attribute>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>541</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AddPosition">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Додати посаду</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDepartment">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Додати департ</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>367</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relogin_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Перезайти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
